### PR TITLE
Add other cmds to main.rs and change helps

### PR DESCRIPTION
**Pull request by xphoniex**

### CODE REVIEW

File: radicle-cli/src/commands/checkout.rs [hunk 1]
```diff
@@ -13,7 +13,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "checkout",
-    description: "Checkout a radicle project working copy",
+    description: "Checkout a project's working copy",
```
Comment (slack-coder) [inline radicle-cli/src/commands/checkout.rs:16]: What about 'checkout a project into the local directory"?  The term "working copy" might not be clear for newcomers to the project.

File: radicle-cli/src/commands/patch.rs [hunk 2]
```diff
@@ -19,7 +19,7 @@ use crate::terminal::patch::Comment;
 
 pub const HELP: Help = Help {
     name: "patch",
-    description: "Work with radicle patches",
+    description: "Work with a patch",
```
Comment (slack-coder) [inline radicle-cli/src/commands/patch.rs:22]: Maybe "manage patches"?  "work with a patch" excludes creating one, is this the case?

File: radicle-cli/src/commands/path.rs [hunk 3]
```diff
@@ -9,7 +9,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "path",
-    description: "Display the radicle home path",
+    description: "Display the home path",
```
Comment (slack-coder) [inline radicle-cli/src/commands/path.rs:12]: referring to 'radicle' here may make sense.  The term 'home path' is to close to the more understood user 'home path'

File: radicle-cli/src/commands/review.rs [hunk 4]
```diff
@@ -14,7 +14,7 @@ use crate::terminal::patch::Comment;
 
 pub const HELP: Help = Help {
     name: "review",
-    description: "Approve and reject radicle patches",
+    description: "Approve and reject a patche",
```
Comment (slack-coder) [inline radicle-cli/src/commands/review.rs:17]: typo ;)

**Comment by slack-coder** [comment]
otherwise :+1:

File: radicle-cli/src/commands/path.rs (hunk 3 above)
Comment (cloudhead) [reply radicle-cli/src/commands/path.rs:12]: Agreed.

File: radicle-cli/src/commands/track.rs [hunk 5]
```diff
@@ -12,7 +12,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "track",
-    description: "Manage radicle project tracking relationships",
+    description: "Manage project's tracking relationships",
```
Comment (cloudhead) [inline radicle-cli/src/commands/track.rs:15]:
```suggestion
    description: "Manage project tracking relationships",
```

File: radicle-cli/src/commands/untrack.rs [hunk 6]
```diff
@@ -12,7 +12,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "untrack",
-    description: "Untrack radicle project peers",
+    description: "Untrack project's peers",
```
Comment (cloudhead) [inline radicle-cli/src/commands/untrack.rs:15]:
```suggestion
    description: "Untrack project peers",
```

**Comment by xphoniex** [comment]
Applied the requested changes.

**Review by cloudhead — APPROVED** [review]
Thanks!

**Pull request merged**